File: BIM(2nd Sem)/Test/prime.java
class prime{
    public static void main(String[]args){
        int n=5,flag=1;
        for(int i=2;i<n;i++){
            if(n%i==0){
                flag=0;
                break;
            }
        }
        if(flag==0){
            System.out.println(n+" is not a prime number");
        }
        else{
            System.out.println(n+" is a prime number");
        }
    }
}

File: BIM_Lab_Work/BIM(1st Sem)/Practise/prime.c
#include<stdio.h>

void main() {
	int a[100],i,j,n,flag=1;
	printf("How many values do you want to enter? ");
	scanf("%d",&n);
	printf("Enter %d values: ",n);
	for(i=0;i<n;i++) {
		scanf("%d",&a[i]);
	}
	printf("Prime numbers in the array: \n");
	for(i=0;i<n;i++){
		flag=1;
		if(a[i]<=1){
			continue;
		}
		for(j=2;j<a[i];j++){
			if(a[i]%j==0){
				flag=0;
				break;
			}
		}
		if(flag){
			printf("%d\n",a[i]);
		}
	}	
}
